components/Dropdown/Dropdown.jsx

```diff
@@ -212,14 +216,12 @@ const SelectedItemLabel = styled.span`
 `;
 
 const DropItemImage = styled.img`
-  float: right;
   width: ${DROPITEM_IMAGE_SIZE};
   height: ${DROPITEM_IMAGE_SIZE};
+  margin-left: ${spacing.xsmall}px;
 `;
 
-const DropItemLabel = styled.span`
-  vertical-align: middle;
-`;
+const DropItemLabel = styled.span``;
```

Comment:
@alisson-suzigan what do you think to remove this component and use `span` direct in `            <DropItemLabel>{_getLabel(item)}</DropItemLabel>`? It was removed `vertical-align: middle` style, is not necessary create this component.

Reply:
i agree with this.

Reply:
Yes, that makes sense 🔍  good eyes.
I just submitted the change.